Makefile

```diff
@@ -53,47 +75,54 @@ VECTORTILES_URL ?= //vectortiles{s}.geo.admin.ch
 VECTORTILES_TECH_URL ?= //vectortiles{s}.
 LAST_VECTORTILES_URL := $(call lastvalue,vectortiles-url)
 
-PUBLIC_URL_REGEXP ?= ^https?:\/\/public\..*\.(bgdi|admin)\.ch\/.*
-ADMIN_URL_REGEXP ?= ^(ftp|http|https):\/\/(.*(\.bgdi|\.geo\.admin)\.ch)
-HREF_REGEXP ?= ^\s*(https?|whatsapp|file|s?ftp|blob|mailto):
 
 # E2E variables
 E2E_TARGETURL ?= https://mf-geoadmin3.dev.bgdi.ch
 E2E_TESTS ?= false
 E2E_BROWSER ?= false
 E2E_SINGLE ?= false
+SAUCELABS_TESTS ?=
 
+
+# Less variables
 LESS_PARAMETERS ?= -ru
 
-GIT_BRANCH ?= $(shell if [ -f .build-artefacts/deployed-git-branch ]; then cat .build-artefacts/deployed-git-branch 2> /dev/null; else git rev-parse --symbolic-full-name --abbrev-ref HEAD; fi)
-GIT_LAST_BRANCH := $(call lastvalue,git-branch)
-BRANCH_TO_DELETE ?=
+
+# Map libs variables
 OL_VERSION ?= v4.6.5 # v4.6.5, March 20 2018
 OL_CESIUM_VERSION ?= 071c090db82540fb3c9cbae9e04699e783ca8203 # master, March 27 2018
 CESIUM_VERSION ?= dabbd96f70f568aeae6eed367d52b8546458f948 # c2c/c2c_patches, March 2 2018
-DEFAULT_TOPIC_ID ?= ech
+
+
+# App variables
 TRANSLATION_FALLBACK_CODE ?= de
 LANGUAGES ?= '[\"de\", \"fr\", \"it\", \"en\", \"rm\"]'
+DEFAULT_TOPIC_ID ?= ech
+
+
+# Translations variables
 LANGS ?= de fr it rm en
 TRANSLATE_GSPREAD_KEYS ?= 1F3R46w4PODfsbJq7jd79sapy3B7TXhQcYM7SEaccOA0
 TRANSLATE_CSV_FILES ?= "https://docs.google.com/spreadsheets/d/1bRzdX2zwN2VG7LWEdlscrP-wGlp7O46nvrXkQNnFvVY/export?format=csv&gid=54811248"
 TRANSLATE_EMPTY_JSON ?= src/locales/empty.json
 TRANSLATE_OUTPUT ?= src/locales
+
+
+# Map variables
+DEFAULT_EPSG ?= EPSG:2056
+DEFAULT_EPSG_EXTEND ?= '[2420000, 1030000, 2900000, 1350000]'
 DEFAULT_EXTENT ?= '[2420000, 1030000, 2900000, 1350000]'
 DEFAULT_RESOLUTION ?= 500.0
-DEFAULT_LEVEL_OF_DETAIL ?= 7 #level of detail for the default resolution
 RESOLUTIONS ?= '[650.0, 500.0, 250.0, 100.0, 50.0, 20.0, 10.0, 5.0, 2.5, 2.0, 1.0, 0.5, 0.25, 0.1]'
+TILEGRID_ORIGIN ?= '[2420000, 1350000]'
+TILEGRID_RESOLUTIONS ?= '[4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250, 1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5, 0.25, 0.1]'
+
+
+# Globe variables
+DEFAULT_LEVEL_OF_DETAIL ?= 7 #level of detail for the default resolution
```

Comment:
`DEFAULT_LEVEL_OF_DETAIL` is 2500 and  `LEVEL_OF_DETAILS` is 6 from 7/2500 ?

Reply:
default resolution = 500 -> Map zoom = 1 -> Globe lod = 7